ScrumPoker.Data/Data/GameRoomDataBase.cs

```diff
@@ -0,0 +1,47 @@
+using ScrumPoker.Core.Models;
+using ScrumPoker.DataBase.Interfaces;
+
+namespace ScrumPoker.Data.Data;
+
+public class GameRoomDataBase : IGameRoomData
+{
+    private static List<GameRoom> _gameRooms = new List<GameRoom>();
+    private static int Id { get; set; }
+    
+    public void Create(string name)
```

Comment:
Pass the object here, when `GameRoom` has more properties you will need to create object in DB with all those properties

Reply:
Also return the newly created object